ci-git-release.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -eo pipefail
```

Comment:
Add comment on what script does.

Reply:
How/when is this script used?

Reply:
I no longer use this script... as all the logic is now part of the circleci config.yml

I could delete it